words.yml

```diff
@@ -1,10 +0,0 @@
-- words:
```

Comment:
I think we could probably have a `misc` category or something, when we reach that point.